daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -110,6 +110,14 @@ metrics:
   rules: []
   http:
     increasedCardinality: true
+    pathMatching:
+      ingress:
+      - /orders/{orderID}
+      - /orders/{orderID}/items/{itemID}
+      egress:
+      - /orders/{orderID}
```

Comment:
I recommend we use a different API, like "payment".

All of your examples, seems that "egress" and "ingress" have a big overlap, sometimes the same list. Can we combine those two and have users to provide a single list?

Reply:
updated according to the proposal update

Reply:
@nelson-parente - Can this be resolved?

Reply:
yes!